### DISCUSSION TITLE
Widget need to show after the content render in blog or article post - Decouple CMS

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#10668] [Q&A]
![widgetissue](https://user-images.githubusercontent.com/5202291/140784098-8fb7126f-7748-4f53-ab4d-547ae7045209.png)

**Comment by ns8482e** [comment]
In `Layout.cshtml` use ` @await RenderBodyAsync()`

**Reply by saleemhaq** [reply]
> @await RenderBodyAsync()

It is still showing the same like earlier? 
![widgetissue](https://user-images.githubusercontent.com/5202291/140797321-6496311d-021e-4069-a014-4f9e57b08606.png)
